Dockerfile

```diff
@@ -1,50 +1,50 @@
-FROM ubuntu:trusty
+FROM isuper/java-oracle:jdk_8
```

Comment:
From my point of view:
- this Dockerfile is to avoid users to install all the stack (java, node, npm, bower, gulp, generator-jhipster), and is already for advanced users
- it is used like a VM, with all classic tools : bash, vim, curl, sudo, etc...
- `ubuntu:trusty` is bigger, but provide all these tools by default
- we can migrate to alpine, but we have to provide all these minimum tools

About the migration to alpine.
Pros:
- very light

Cons:
- for very advanced users, because it is slighty different from ubuntu
- more complex to write Dockerfile -> all links should be official and we shouldn't depend on a private/personnal repo
- need to get all default tools, a user would need (vim, curl, sudo...)

To conclude:
- I personnaly prefer to stay with ubuntu:trusty for this image
- about containerize microservice => java:openjdk-8u72-jre (it is our current base image) or alpine
- we could have different Dockerfile :-) but not sure to put them all in the root folder...